### PR TITLE
Add update service method

### PR DESCRIPTION
**Pull request by mpguerra**
To support corresponding change in 3scale-toolbox to update existing services when doing a copy

### CODE REVIEW

File: lib/3scale/api/client.rb [hunk 1]
```diff
@@ -103,6 +103,15 @@ def create_service(attributes)
 
       # @api public
       # @return [Hash]
+      # @param [Hash] attributes Service Attributes
+      # @option attributes [String] :name Service Name
+      def update_service(attributes, service_id)
+        response = http_client.put("/admin/api/services/#{service_id}", body: { service: attributes })
```
Comment (mikz) [inline lib/3scale/api/client.rb]: I think for update would be great to use the new `PATCH` method.
And if it is is missing in the http client, add it there.

**Comment by mikz** [comment]
There are few things missing. 

We have to have everything covered by test. There are two kinds of test: unit and integration.
The unit tests are there: https://github.com/3scale/3scale-api-ruby/blob/master/spec/api/client_spec.rb
They don't need internet, just test that you call the http client with proper attributes etc.

The integration tests are there: https://github.com/3scale/3scale-api-ruby/blob/master/spec/integration/service_spec.rb
and they require credentials to 3scale portal, because they actually do live calls.
Which environment variables are needed is on the top of the file.


And last thing would be adding a changelog entry.

File: lib/3scale/api/client.rb (hunk 1 above)
Comment (mayorova) [inline lib/3scale/api/client.rb]: The documented endpoint for Service Update is using `PUT`:
screen shot 2017-07-10 at 18 20 02

Comment (mikz) [inline lib/3scale/api/client.rb]: Yes, and Rails accepts both PUT and PATCH as equal.

**Pull request merged**